src/modules/network/implementation/libp2p-service.js

```diff
@@ -632,12 +632,16 @@ class Libp2pService {
     }
 
     removeCachedSession(operationId, keywordUuid, peerIdString) {
-        if (this.sessions[peerIdString]?.[operationId]?.[keywordUuid]?.stream) {
-            this.sessions[peerIdString][operationId][keywordUuid].stream.close();
-            delete this.sessions[peerIdString][operationId];
-            this.logger.trace(
-                `Removed session for remotePeerId: ${peerIdString}, operationId: ${operationId}.`,
-            );
+        try {
```

Comment:
it looks to me like only the closing of the stream can fail here, so it makes sense to only try / catch that line

Reply:
thanks, makes sesne so i changed as you suggested